slick.grid.js

```diff
@@ -5827,6 +5842,7 @@ if (typeof Slick === "undefined") {
       "getTopPanel": getTopPanel,
       "setTopPanelVisibility": setTopPanelVisibility,
       "getPreHeaderPanel": getPreHeaderPanel,
+      "getPreHeaderPanelR": getPreHeaderPanelR,
```

Comment:
Can you change the public exposed method to be more meaningful, `getPreHeaderPanelRight` is probably better. We probably don't have the left side, or at least not in the name, but I wonder if we should add an alias for `getPreHeaderPanelLeft` wich would return `getPreHeaderPanel`.

Reply:
Makes sense, yeah. It's more intuitive to have the api explicitly identify the left and right functions when working with the frozen columns.